config/jobs/gardener-extension-shoot-rsyslog-relp/gardener-extension-shoot-rsyslog-relp-integration-tests.yaml

```diff
@@ -0,0 +1,59 @@
+presubmits:
+  gardener/gardener-extension-shoot-rsyslog-relp:
+  - name: pull-gardener-extensioon-shoot-rsyslog-relp-integration
+    cluster: gardener-prow-build
+    always_run: true
+    skip_branches:
+    - release-v\d+.\d+ # don't run on release branches for now (add a job per branch later)
+    decorate: true
+    decoration_config:
+      timeout: 20m
+      grace_period: 10m
+    annotations:
+      description: Runs integration tests for Gardener extension shoot-rsyslog-relp developments in pull requests
+      fork-per-release: "true"
```

Comment:
we use `fork-per-release` to create copies of the jobs when a new release branch was created. This is probably not required for `gardener-extension-shoot-rsyslog-relp` repo.